editors/code/src/extension.ts

```diff
@@ -129,3 +135,43 @@ export function deactivate(): Thenable<void> {
     }
     return Server.client.stop();
 }
+
+async function askToCargoWatch() {
+    const watch = await vscode.window.showInformationMessage(
```

Comment:
I think we should have a config option here to suppress the notification.

Reply:
So should this be entirely config-controlled? Either you have it enabled there and you are not prompted or you disable it completely or:
- enabled, so we ask you everytime
- disabled, so we never ask you?

Reply:
Implemented the "when you explicitly turn off a config option we won't bother you anymore" option in https://github.com/rust-analyzer/rust-analyzer/pull/995/commits/7c2595c26820917fa9ad1b1c36f01fc6ac979287